datacommons/setup.py

```diff
@@ -23,15 +25,10 @@
 URL = 'https://github.com/datacommonsorg/api-python'
 EMAIL = 'support@datacommons.org'
 AUTHOR = 'datacommons.org'
-REQUIRES_PYTHON = '>=2.7'
+REQUIRES_PYTHON = '>=3.7'
```

Comment:
Do we know if anyone is still using python 2.7 and should be alerted about this change?

Reply:
The change will be pushed as a new pip version. So users with older version can still use python 2.7